internal/controller/api/v1beta1/agent_types.go

```diff
@@ -222,6 +222,9 @@ type DebugInfo struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=".spec.clusterDeploymentName.name",description="The name of the cluster the Agent registered to."
 // +kubebuilder:printcolumn:name="Approved",type="boolean",JSONPath=".spec.approved",description="The `Approve` state of the Agent."
+// +kubebuilder:printcolumn:name="Role",type="string",JSONPath=".status.role",description="The role (master/worker) of the Agent."
+// +kubebuilder:printcolumn:name="Stage",type="string",JSONPath=".status.progress.currentStage",description="The HostStage of the Agent."
+// +kubebuilder:printcolumn:name="Hostname",type="string",JSONPath=".status.inventory.hostname",description="The hostname of the Agent."
```

Comment:
Maybe we should have both the hostname from the inventory and the hostname from the spec?
`Requested Hostname` maybe?

Reply:
yeah, we can add it if useful. though we have quite many columns now, so probably should have it in priority, so would be available only using wide output.